features/login-testcase.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login testcase. 
+
+    Positive scenario (passing valid login and password).
+
+    Scenario: User opens pah-fm website.
+       Given User has account in the system already created by system admin.
```

Comment:
I am mostly familiar with the first-person-perspective (Given I am on X website, When I click something, Then the result is visible), although this is not wrong. Please be aware, that "I" notation is most commonly used.